libraries/commerce/category/selectors/index.js

```diff
@@ -105,6 +105,14 @@ export const getCurrentCategoryChildCount = createSelector(
   category => (category ? category.childrenCount : null)
 );
 
+/**
+ * @type {number}
```

Comment:
What does that mean? What does it take in? What does it return? Please correct that.